src/routes/profile/data.ts

```diff
@@ -26,25 +26,29 @@ export async function deleteProfileById(database: D1Database, profileId: string)
 }
 
 interface InsertProfileParams {
-	payload: Pick<Profile, 'description' | 'email' | 'links' | 'name'>;
+	payload: Pick<
+		Profile,
+		'description' | 'email' | 'links' | 'name' | 'password'
+	>;
 	database: D1Database;
 }
 
 export async function insertProfile({
-	payload: { email, name, description, links },
+	payload: { email, name, password, description, links },
```

Comment:
Unrelated to this specific line but I think this file now leaks password hashes because `SELECT *` was used. Generally a good idea to avoid `SELECT *`. Alternatively you could move the password in a separate table.

Reply:
You mean having a table with only ids and passwords? That sounds like a good idea...

Reply:
yeah it helps to compartmentalize a bit! It's not the worst to have it in the `users` table but the problem is that objects are passed through end-to-end and with `SELECT *` which is a bit dangerous. Ideally you explicitly pass each property at each layer (request body -> controller -> service -> controller -> response body)